plugins/modules/system/open_iscsi.py

```diff
@@ -57,6 +57,11 @@
         - Whether the target node should be automatically connected at startup.
         type: bool
         aliases: [ automatic ]
+    auto_portal_startup:
+        description:
+        - Whether the target node portal should be automatically connected at startup.
```

Comment:
Can you add an example using this option?

Reply:
added in commit _be951f3_